content/rs/clusters/monitoring/uptrace-integration.md

```diff
@@ -0,0 +1,185 @@
+
+```shell
+docker-compose logs otel-collector
+
+# or
+
+sudo journalctl -u otelcol-contrib -f
+```
+
+You can also check the full OpenTelemetry Collector config [here](https://github.com/uptrace/uptrace/blob/master/example/redis-enterprise/otel-collector.yaml).
+
+## Viewing metrics
+
+When metrics start arriving to Uptrace, you should see a couple of dashboards in the Metrics tab. In total, Uptrace should create 3 dashboards for Redis Enterprise metrics:
+
+- "Redis: Nodes" dashboard displays a list of cluster nodes with ability to click on a node to view node metrics.
+
+- "Redis: Databases" displays a list of Redis databases in all cluster nodes. To find a specific database, you can use filters or sort the table by columns.
+
+- "Redis: Shards" contains a list of shards that you have in all cluster nodes. Just like with other dashboards, you can filter/sort shards and click on a shard to get more details.
+
+## Monitoring metrics
+
+Uptrace can also monitor metrics and send notifications using AlertManager integration. First, you need to tell Uptrace how to reach AlertManager:
+
+```yaml
+# uptrace.yml
+
+##
+## AlertManager client configuration.
+## See https://uptrace.dev/get/alerting.html for details.
+##
+## Note that this is NOT an AlertManager config and you need to configure AlertManager separately.
+## See https://prometheus.io/docs/alerting/latest/configuration/ for details.
+##
+alertmanager_client:
+  # AlertManager API endpoints that Uptrace uses to manage alerts.
+  urls:
+    - "http://localhost:9093/api/v2/alerts"
+```
+
+To start monitoring metrics, you need to add some alerting rules, for example, to alert when a Redis shard is down:
+
+```yaml
+# uptrace.yml
+
+alerting:
+  rules:
+    - name: Redis shard is down
+      metrics:
+        - redis_up as $redis_up
+      query:
+        - group by cluster
+        - group by bdb
+        - group by node
+        - $redis_up == 0
+      for: 5m # for 5 minutes
+      projects: [1] # in projects #1
+```
+
+You can also create queries with more complex expressions, for example, to alert when the keyspace hit rate is lower than 75% or memory fragmentation is too high:
+
+```yaml
+# uptrace.yml
+
+alerting:
+  rules:
+    - name: Redis read hit rate < 75%
+      metrics:
+        - redis_keyspace_read_hits as $hits
+        - redis_keyspace_read_misses as $misses
+      query:
+        - group by cluster
+        - group by bdb
+        - group by node
+        - $hits / ($hits + $misses) < 0.75
+      for: 5m
+      projects: [1]
+
+    - name: Memory fragmentation is too high
+      metrics:
+        - redis_used_memory as $mem_used
+        - redis_mem_fragmentation_ratio as $fragmentation
+      query:
+        - group by cluster
+        - group by bdb
+        - group by node
+        - $mem_used > 32mb and $fragmentation > 3
+      for: 5m
+      projects: [1]
+```
+
+## Conclusion
+
+TODO: some final words and links to the blog etc
```

Comment:
I think it would be okay to remove the conclusion section unless you want to replace it with a **More info** section and add a few useful links.
```suggestion
```